README.md

```
@@ -103,6 +105,29 @@ npx @capgo/cli@latest init YOUR_API_KEY com.example.app
+
+📱 Test the same Capacitor device target picker used by init onboarding.
+For iOS, this asks whether to use a physical iPhone/iPad or simulator, supports checking again for targets, and runs with the resolved target when available.
+Use --no-launch to print the resolved command without starting the app.
+
+**Example:**
+
+```bash
+npx @capgo/cli@latest run-device ios --no-launch
+```
+
+## <a id="options"></a> Options
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)